src/i18n/translations/en.ts

```diff
@@ -650,4 +650,10 @@ export const messages: MessagesKey = {
   mapLoadingStopsDataText: 'Loading stops...',
   locateStopPoint: 'Locate',
   locateStopPointTooltip: 'Locate on map',
+  exportsDownloadUnsupportedBrowserTitle: 'Browser Not Supported',
+  exportsDownloadUnsupportedBrowserMessage:
+    'Sorry, your browser does not support file downloads.',
+  exportsDownloadErrorTitle: 'Download Failed',
+  exportsDownloadErrorMessage:
+    'An error occurred while downloading the export. Please try again.',
```

Comment:
Swedish: Ett fel har uppstått medan filen laddades ner. Försök igen.

Reply:
Finnish: _Latauksen aikana tapahtui odottamaton virhe. Yritä uudelleen._

Literally "During download an unexpected error occurred. Try again." Finnish isn't naturally polite, we could start the sentence with _Valitettavasti latauksen..._ ("Sadly", "Unfortunately", "Regrettably", "Lamentably") but that's maybe a bit too soft for this. Plus if the sentence starts softly, then the ending should also reflect that with _Yritäthän_ ("Please try...") or _Yrittäisitkö_ ("Could you try...") but that's starts to sound kinda condescending and makes this translation a bit long.

Agglutinative languages, yay? :)